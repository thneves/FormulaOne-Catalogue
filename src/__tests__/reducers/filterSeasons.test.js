import filterSeasonReducer from '../../redux/reducers/filterSeason';

describe('Filter Seasons Reducer', () => {
  it('should return initial state', () => {
    expect(filterSeasonReducer(undefined, {})).toMatchSnapshot();
  });

  it('should handle add a new filter', () => {
    expect(filterSeasonReducer([],
      {
        type: 'ADD_FILTER',
      })).toMatchSnapshot();
  });

  it('should handle remove filters', () => {
    expect(filterSeasonReducer([],
      {
        type: 'REMOVE_FILTER',
      })).toMatchSnapshot();
  });

  it('should handle reset filter', () => {
    expect(filterSeasonReducer([],
      {
        type: 'RESET_FILTER',
      })).toMatchSnapshot();
  });
});
