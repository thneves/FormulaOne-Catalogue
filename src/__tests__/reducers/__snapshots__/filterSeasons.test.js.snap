// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter Seasons Reducer should handle add a new filter 1`] = `
Array [
  undefined,
]
`;

exports[`Filter Seasons Reducer should handle remove filters 1`] = `Array []`;

exports[`Filter Seasons Reducer should handle reset filter 1`] = `Array []`;

exports[`Filter Seasons Reducer should return initial state 1`] = `Array []`;
